modules/configuring-layer-three-routed-topology.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-layer-three-routed-topology_{context}"]
+= Configuring a routed topology
+
+The routed (_layer 3_) topology networks are a simplified topology for the cluster default network without egress. In this topology, there is a logical switch per node, each with a different subnet, and a router interconnecting all logical switches.
+
+[NOTE]
+====
+Layer 3 routed topology networks only allow for the transfer of data packets from server to server within a data center.
+====
+
+The following `NetworkAttachmentDefinition` custom resource definition (CRD) YAML describes the fields needed to configure a routed secondary network.
+
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l3-network
+  namespace: ns1
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1",
+            "name": "l3-network", <1>
+            "type": "ovn-k8s-cni-overlay", <2>
+            "topology":"layer3", <3>
+            "subnets": "10.128.0.0/16/24", <4>
+            "mtu": 1300, <5>
+            "netAttachDefName": "ns1/l3-network" <6>
+    }
+----
+<1> The name of the network. This attribute is not namespaced.
+<2> `ovn-k8s-cni-overlay` is required as the config type.
+<3> `layer3` is required as the config topology when setting a layer 3 routed topology.
+<4> A comma separate list of subnets. When multiple subnets are provided, the user gets an IP from each subnet. This attribute indicates both the subnet across the cluster, and per node. In this example, there is a `/16` subnet for the network, but each node has a `/24` subnet.
+<5> Optional. Sets MTU to the specified value. If unspecified, defaults to the value chosen by the kernel.
+<6> Must match `/` of the surrounding object.
```

Comment:
I don't understand this explanation. Maybe I'm just missing something ?

Reply:
No, I meant the part about the
> Must match `/` of the surrounding object.

I know what happened: we failed to comply with the markdown renderer! 

If you look at the [raw document](https://raw.githubusercontent.com/ovn-org/ovn-kubernetes/master/docs/multi-homing.md) you'll see it says:
> `netAttachDefName` (string, required): must match `<namespace>/<net-attach-def name>` of the surrounding object.